nft_ingester/src/bin/slot_persister/main.rs

```diff
@@ -189,9 +190,24 @@ pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
     info!("Starting Slot persister...");
 
     let args = Args::parse();
+
     let mut metrics_state = MetricState::new();
     metrics_state.register_metrics();
 
+    let rpc_client = Arc::new(BackfillRPC::connect(args.rpc_host));
+
+    let backfill_source = {
+        if let Some(bg_creds) = args.big_table_credentials {
+            Arc::new(BackfillSource::Bigtable(Arc::new(
+                BigTableClient::connect_new_with(bg_creds, args.big_table_timeout)
+                    .await
+                    .expect("expected to connect to big table"),
+            )))
+        } else {
+            Arc::new(BackfillSource::Rpc(rpc_client.clone()))
+        }
+    };
```

Comment:
please move back after the ctrl+c handler, so the declaration is not far from the usage